### PR TITLE
feat: add label sync and label list

### PR DESCRIPTION
**Pull request by daviddias**
Related to https://github.com/filecoin-project/lotus/pull/2629

Can folks confirm if I got all the "Area" labels right? Which ones am I missing?

### CODE REVIEW

File: .github/labels.yml [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+#
+- name: "good first issue"
+  color: 7057ff
+  description: "Good for newcomers"
+- name: "help wanted"
+  color: 008672
+  description: "Extra attention is needed"
+
+###
+### Areas
```
Comment (magik6k) [inline .github/labels.yml:13]: I'd say:
* chain/vm
* chain/sync
* chain[/misc]
* sealing/fsm
* sealing/storage
* proving
* mining
* client/storage
* client/retrieval
* wallet
* paych
* multisig
* networking

(looks like `description:`s aren't matching)

Comment (daviddias) [reply .github/labels.yml:13]: Thanks! This is what I needed!

Comment (daviddias) [reply .github/labels.yml:13]: done

**Comment by daviddias** [comment]
> Were you planning on using "needs analysis" as "someone's gotta investigate this"? 
Exactly, it suggests that the issue needs investigation so that it is flushed out to be actionable (typically meaning that someone more senior needs to spelunk to come to a conclusion before someone less senior can take it on)

**Pull request merged**